data/food/export_agb/export_builder.py

```diff
@@ -224,6 +208,27 @@ def search(name):
         outfile.write("\n")
     print(f"\nExported {len(ingredients)} ingredients to {INGREDIENTS}")
 
+    # display impacts that have changed
+    old = {p["id"]: p["impacts"] for p in oldbuilder}
+    review = False
+    for p in builder:
+        for impact in builder[p]["impacts"]:
+            if old.get(p, {}).get(impact, {}):
+                percent_change = (
+                    100
+                    * abs(builder[p]["impacts"][impact] - old[p][impact])
+                    / old[p][impact]
+                )
+                if percent_change > 0.1:
```

Comment:
How about making this `0.1` value a command line parameter? 🤔

Reply:
it does display all differences of more than 0.1% so it'll catch differences of more than 3% or 5%

Reply:
Though that could create lots of entries (noise), right? Anyway, I'm just trying to be helpful here, your call.